iron-overlay-behavior.html

```diff
@@ -155,11 +155,8 @@
       return Polymer.dom(this).querySelector('[autofocus]') || this;
     },
 
-    registered: function() {
-      this._backdrop = document.createElement('iron-overlay-backdrop');
```

Comment:
this was the suggestion that causes the tests to fail :x
Never mind it, the import should be done here, in `iron-overlay-behavior` since in here we use the methods of `iron-overlay-backdrop`, and should not be done in `iron-overlay-manager`